src/cmdstan/stansummary_helper.hpp

```diff
@@ -86,8 +87,7 @@ using Eigen::Dynamic;
 
 Eigen::VectorXi calculate_column_widths(
     const Eigen::MatrixXd &values, const std::vector<std::string> &headers,
-    const int sig_figs,
-    Eigen::Matrix<std::ios_base::fmtflags, Dynamic, 1> &formats) {
+    int sig_figs, Eigen::Matrix<std::ios_base::fmtflags, Dynamic, 1> &formats) {
```

Comment:
Any reason some of these went from const to non-const?

Reply:
specifying an integer argument value as `const` doesn't mean anything - it's passed by value anyhow.
changed this so that all API calls follow this practice.

Reply:
As long as we are consistent one way or another I'm fine with it